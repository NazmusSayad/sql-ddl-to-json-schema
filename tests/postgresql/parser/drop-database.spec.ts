import { getParsedFormat } from '../parse-handler';
import { run } from '../runner';

run(getParsedFormat, {
  'Parser: Should drop test database': {
    queries: [
      'DROP DATABASE test;',
      'drop database test;',
      'drop SCHEMA `test`;',
      `drop
        schema
        test
      ;`,
    ],
  },
});
