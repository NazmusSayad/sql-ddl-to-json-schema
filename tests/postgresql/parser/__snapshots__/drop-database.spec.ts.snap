// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Parser: Should drop test database 1`] = `
ParsedQuerySerializer {
  "parsed": {
    "def": [
      {
        "def": {
          "def": "test",
          "id": "P_DROP_DB",
        },
        "id": "P_DDS",
      },
    ],
    "id": "MAIN",
  },
}
`;

exports[`Parser: Should drop test database 2`] = `
ParsedQuerySerializer {
  "parsed": {
    "def": [
      {
        "def": {
          "def": "test",
          "id": "P_DROP_DB",
        },
        "id": "P_DDS",
      },
    ],
    "id": "MAIN",
  },
}
`;

exports[`Parser: Should drop test database 3`] = `
ParsedQuerySerializer {
  "parsed": {
    "def": [
      {
        "def": {
          "def": "test",
          "id": "P_DROP_DB",
        },
        "id": "P_DDS",
      },
    ],
    "id": "MAIN",
  },
}
`;

exports[`Parser: Should drop test database 4`] = `
ParsedQuerySerializer {
  "parsed": {
    "def": [
      {
        "def": {
          "def": "test",
          "id": "P_DROP_DB",
        },
        "id": "P_DDS",
      },
    ],
    "id": "MAIN",
  },
}
`;
